### ISSUE TITLE
Sound stutter/crackling on iOS 13.1.3, iPhone XS Max, Chrome

### ISSUE THREAD

**Issue by Joohansson** [Tonejs/Tone.js#558]
I have spent days trying to solve this. The sound is good on my PC with Chrome (or even a simple windows tablet) but not on my iPhone. I reduced my app as far as possible and made a very small demo with just a simple noise generator for you to replicate. You find the code by checking the page source. It sounds really bad, like a distorted sound lag. If you can't hear it, then it works on your device. Sometimes you have to listen for a minute before the crackling starts.

I think this is true for any type of semi-complicated sound with tone.js. Is it a performance issue? This phone should be able to handle much worse applications. Is there anything I can do?

**Demo:** https://nanoticker.info/tone-demo/

**Comment by tambien** [comment]
both demos include Tone.Freeverb which i think may be the culprit here. if you have a second could you try this out with Tone.Reverb instead?

**Comment by Joohansson** [comment]
That's actually helpful. I replaced freeverb with reverb and even if the sound was not exactly the same it solved the stutter in both demos. However, I have updated the noise demo with a polysynth that is also in my real app. And as soon as I add that, the stutter is back. That is the polySynth.triggerAttackRelease in the loop function.

It's like the phone can't handle when there is too much going on at the same time. Or you will find some other bad stuff in my code. Much appreciated.
